src/pages/events/project-setup.md

```diff
@@ -1,58 +1,179 @@
 
-To create a project:
+## Set up a project
 
-1. In the [Adobe Developer Console], select the desired organization from the dropdown in the top-right corner.
+[Projects Overview](https://developer.adobe.com/developer-console/docs/guides/projects/) describes the different types of projects and how to manage them. Here, we'll create a templated project.
 
-1. Select an existing project or [create a new project].
+1. In the Adobe Developer Console, select the desired organization from the dropdown menu in the top-right corner.
 
-    ![create a project](../_images/create-project.png)
+1. Select **Create project from template**.
 
-1. Inside the project, click **Add API**.
+   ![Create a project](../_images/create-project.png)
 
-    ![add an api](../_images/add-api.png)
+1. On the **Set up templated project** page, specify a project title and app name. Then click **Save**. The Console creates a Production and a Stage workspace.
 
-1. Filter by Adobe Services, select **I/O Events**, and click **Next**.
+   ![Add an api](../_images/set-up-templated-project.png)
 
-    ![select io management api](../_images/adobe-io-events.png)
+1. In the Stage workspace, click the **Add service** pop-up menu and select **API**.
 
-1. Select the **Generate a key pair** option and click **Generate keypair**. A `config.zip` file, which contains a `certificate_pub.crt` and `private.key` file, will automatically download.
+1. On the **Add an API** page, filter by Adobe Services, select **I/O Events**, and click **Next**.
 
-    ![generate a key pair](../_images/generate-key-pair.png)
+   ![Select IO events](../_images/adobe-io-events.png)
 
-    **Note**: If you want to manually [create a public key certificate], you can select the **Upload your public key** option.
+1. Select the **Generate a key pair** option and click **Generate keypair**. The `config.zip` file downloads automatically.
+
+   ![generate a key pair](../_images/generate-key-pair.png)
+
+   **Note**: If you want to manually [create a public key certificate](https://developer.adobe.com/developer-console/docs/guides/authentication/JWT/JWTCertificate/), you can select the **Upload your public key** option.
 
 1. Click **Save configured API**.
 
-## Download workspace configuration
+1. Unzip the downloaded `config.zip` file. The extracted `config` directory should contain a `certificate_pub.crt` and a `private.key` file.
+
+## Download the workspace configuration file
 
 To download a `.json` file containing your workspace configuration:
 
-1. Open the corresponding project on the [Adobe Developer Console].
+1. Open the corresponding project on the Adobe Developer Console.
 
 1. Select the **Project Overview** and click the **Download** button.
 
-    ![download the workspace config](../_images/download-workspace-config.png)
+   ![download the workspace config](../_images/download-workspace-config.png)
+
+   The `<Workspace-name>.json` file downloads automatically.
```

Comment:
In projects like [Project 49](https://developer.adobe.com/console/projects/283976/4566206088344829845/overview), I'm not seeing that **Download** button. When I select a workspace, I see a **Download all** button that will download the configuration file though. 

I do see that Project 44 has the **Download** button, haven't yet been able to figure out why that is...